Core.Auditing/IfxAuditLogger.cs

```diff
@@ -1,101 +1,124 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 
-using Microsoft.Cloud.InstrumentationFramework;
 using Microsoft.CloudMine.Core.Telemetry;
+using Microsoft.Extensions.Logging;
+using OpenTelemetry.Audit.Geneva;
 using System;
 using System.Collections.Generic;
 using System.Net;
 using System.Net.Sockets;
 
 namespace Microsoft.CloudMine.Core.Auditing
 {
+
+    public class TargetResource
+    {
+        public string type;
+        public string name;
+        public string cluster;
+        public string region;
+
+        public TargetResource(string type, string name, string cluster = null, string region = null)
+        {
+            this.type = type;
+            this.name = name;
+            this.cluster = cluster;
+            this.region = region;
+        }
+    }
+
+    public class CallerIdentity
+    {
+        public CallerIdentityType type;
+        public string name;
+        public string description;
+
+        public CallerIdentity(CallerIdentityType type, string name, string description = null)
+        {
+            this.type = type;
+            this.name = name;
+            this.description = description;
+        }
+    }
+
     public class IfxAuditLogger : IAuditLogger
```

Comment:
Would it be a good idea to rename IfxAuditLogger to AuditLogger?

Reply:
How about `OpenTelemetryAuditLogger`? I think this gives a good indication that we are using OT to implement the `IAuditLogger` interface.

Reply:
I like that